public/sass/components/_panel_header.scss

```diff
@@ -60,6 +60,8 @@ $panel-header-no-title-zindex: 1;
 }
 
 .panel-menu-toggle {
+  position: absolute;
+  top: calc(50% - 9px);
```

Comment:
Not a fan of this solution, but I don't know a better way of doing it. I noticed we are doing something like this already if you look a few lines up on this file. So I don't expect this to be a problem unless in the future the icon size changes

Reply:
not sure the calc is needed here, since you still need a magic pixel count

Reply:
The magic number `9px` comes from `icon_height/2 + icon_y_margin` where `icon_height = 16px` and `icon_y_margin = 2px`. The container element (`.panel-title`) currently has a height of `28px`. So I could replace the `calc` by just `5px` but in the container changes its height in the future the magic number `5px` will not work anymore.

Let me know how should I proceed